GRIB2_Template_4_113_ProductDefinitionTemplate_en.csv

```diff
@@ -0,0 +1,25 @@
+Title_en,OctetNo,Contents_en,Note_en,noteIDs,codeTable,flagTable,Status
+Generalised tiles at a horizontal level or horizontal layer at a point in time,10,Parameter category,(see Code table 4.1),,4.1,,Operational
+Generalised tiles at a horizontal level or horizontal layer at a point in time,11,Parameter number,(see Code table 4.2),,4.2,,Operational
+Generalised tiles at a horizontal level or horizontal layer at a point in time,12,Tile classification,(see Code table 4.242),,4.242,,Operational
+Generalised tiles at a horizontal level or horizontal layer at a point in time,13-14,Type of Tile,(see code table 4.251),,4.251,,Operational
```

Comment:
Should the Description match the Code table name? 

"Type of tile" != "Tile classes and groupings"

Reply:
Do we strictly require this to match perfectly?

Reply:
@amilan17, this seem to be the "remaining question". But I also don't know, if there is a strict rule. Maybe we should discuss this in September.

Reply:
I don't think it can be a requirement. for instance for first/second fixed surfaces, they both refer to the same table type of fixed surface, without mention to "first" or "second"